.stats.yml

```diff
@@ -0,0 +1,4 @@
+configured_endpoints: 801
+openapi_spec_url: https://storage.googleapis.com/stainless-sdk-openapi-specs/telnyx%2Ftelnyx-9b2e42d595d8704628cb792563adef970f306e60f81b682f41747f0e44321d2a.yml
```

Comment:
this is a hashed version of the specs. Will this be updated on every release?

Reply:
yes